### PR TITLE
add more logging around oidc login

### PR DESCRIPTION
**Pull request by rebstar6**
Fixes #.

Changes in this pull request:
-
-

### CODE REVIEW

File: modules/portal/app/controllers/OidcAuthenticator.scala [hunk 1]
```diff
@@ -105,31 +104,34 @@ class OidcAuthenticator @Inject()(wsClient: WSClient, configuration: Configurati
 
   def getCodeCall: Uri = {
     val nonce = new Nonce()
+    val loginId = UUID.randomUUID().toString
```
Comment (rebstar6) [inline modules/portal/app/controllers/OidcAuthenticator.scala]: adding a unique id to each login, helps track the logs across the request

File: modules/portal/app/controllers/OidcAuthenticator.scala [hunk 2]
```diff
@@ -105,31 +104,34 @@ class OidcAuthenticator @Inject()(wsClient: WSClient, configuration: Configurati
...
+        ErrorResponse(500, errorMessage)
       }
       .flatMap {
         case s: AuthenticationSuccessResponse =>
           val code = Option(s.getAuthorizationCode)
           code match {
             case Some(c) => Right(c)
             case None =>
-              Left(ErrorResponse(500, "No code value in getCodeFromAuthResponse"))
+              Left(ErrorResponse(500, s"No code value in getCodeFromAuthResponse"))
```
Comment (britneywright) [inline modules/portal/app/controllers/OidcAuthenticator.scala]: did you mean to pass in a variable here?

Comment (rebstar6) [reply modules/portal/app/controllers/OidcAuthenticator.scala]: ah, no, i did and then i reverted and the s remained

**Comment by rebstar6** [comment]
codecov lines failing are added logging. i think its ok fyi

File: modules/portal/app/controllers/OidcAuthenticator.scala [hunk 3]
```diff
@@ -203,7 +215,7 @@ class OidcAuthenticator @Inject()(wsClient: WSClient, configuration: Configurati
       case err: TokenErrorResponse =>
         val errorMessage = s"Sign in token error: ${err.getErrorObject.getDescription}"
         Left(ErrorResponse(err.toHTTPResponse.getStatusCode, errorMessage))
-      case _ => Left(ErrorResponse(500, "Unable to parse OIDC token response"))
+      case _ => Left(ErrorResponse(500, s"Unable to parse OIDC token response"))
```
Comment (mitruly) [inline modules/portal/app/controllers/OidcAuthenticator.scala]: Don't really need to add `s` unless interpolating variables.

Comment (rebstar6) [reply modules/portal/app/controllers/OidcAuthenticator.scala]: oops missed that one. yeah same as https://github.com/vinyldns/vinyldns/pull/502#discussion_r258966846, removing

**Pull request merged**